messages/TypescriptEslintStrategy.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
First pass at error messages. Open to rewriting any/all of them.

Reply:
You'll have to change the run.md (https://github.com/forcedotcom/sfdx-scanner/blob/dev/docs/_articles/en/scanner-commands/run.md) to include the new flag in the usage, options and also example.

Reply:
This change is only for run command right? not for list ?

Reply:
It's just for the run command. I submitted a new PR to fix the docs https://github.com/forcedotcom/sfdx-scanner/pull/128